lib/linguist/languages.yml

```diff
@@ -4486,6 +4486,16 @@ MiniD:
   tm_scope: none
   ace_mode: text
   language_id: 231
+MiniScript:
+  type: programming
+  color: "#4B4A56"
+  extensions:
+  - ".gs"
```

Comment:
This is no where near meeting our usage requirements and will block this PR even longer than the lack of popularity of the `.ms` extension indicates.

It's probably best to remove this and users can use an override once this PR is merged.